pontoon/checks/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models as m
+
+from pontoon.base.models import Translation
+
+
+class Check(m.Model):
+    """
+    Store checks performed on translations if they failed.
+
+    Severity of failed checks are expressed by subclasses of this model.
+    """
+    library = m.CharField(
+        max_length=20,
+        choices=(
+            ('p', 'pontoon'),
+            ('tt', 'translate-toolkit'),
+            ('cl', 'compare-locales'),
+        ),
+        db_index=True
+    )
+    message = m.TextField()
+
+    class Meta:
+        abstract = True
+        ordering = ('library', 'message')
```

Comment:
Looking at [bug 1436985](https://bugzilla.mozilla.org/show_bug.cgi?id=1436985) I'm a little concerned about performance implications. Shall we move ordering to the `QuerySet` instead?

Note that ths is just a guess and could be a premature optimization. It might be better to leave this unchanged and just add a comment. In the final stage of implementing checks we'll see how much data do we have and how does it behave and then we can work on optimizations.